docs/installation/micro-rdk-dev.md

```diff
@@ -239,8 +240,12 @@ cargo generate --git https://github.com/viamrobotics/micro-rdk.git
 ```
 
 Select `templates/project` when prompted.
+Give the project a name of your choice.
+Select `esp32` for **MCU**.
+If you wish to configure an `esp32-camera` or a `fake` camera as a component of your machine, select **true** for **include camera module and traits**.
+
+You will be prompted to paste your machine's `viam-server` robot JSON configuration into the terminal, which is the same thing as its machine cloud credentials.
 
-You will be prompted to paste your machine cloud credentials into the terminal.
 To obtain this:
 
 - Navigate to [your new machine's](#create-a-new-machine) page on [the Viam app](https://app.viam.com) and select the **CONFIGURE** tab.
```

Comment:
use this one, it's got the updated button
```
{{<imgproc src="configure/machine-part-info.png" resize="500x" declaredimensions=true alt="Restart button on the machine part info dropdown">}}
```

Reply:
changed